chrome/content/zotero/xpcom/translation/translate.js

```diff
@@ -374,7 +374,10 @@ Zotero.Translate.Sandbox = {
 				
 				var translator = translation.translator[0];
 				translator = typeof translator === "object" ? translator : Zotero.Translators.get(translator);
-				translation._loadTranslator(translator)
+				Zotero.Promise.resolve(translator)
```

Comment:
The idea here is that `Zotero.Translators.get()` in the connector is async? Can you document that?